third_party/.gitignore

```diff
@@ -13,11 +13,10 @@
 # Include folders that have hand-written code (not DEPS).
 !accessibility/
 !canvaskit/
-!ninja/
 !spring_animation/
 !test_shaders/
 !tonic/
 !txt/
 !web_locale_keymap/
-!web_test_fonts/
+!web_test_font/
```

Comment:
Looks like the removal of the `s` is a typo?

Reply:
Yup! Nice catch, thanks.